packages/account/src/Configs/personal-details-config.ts

```diff
@@ -190,6 +187,21 @@ export const personal_details_config = ({
     };
 
     const getConfig = () => {
+        // Need to check if client is high risk for MF, if so, remove all fields except first name, last name, date of birth and phone
+        // No need to get tax information when client is high risk for MF
```

Comment:
We are checking is high risk for MT5 right? Could you please check the description here? It says MF.

Reply:
That's correct as the flag is based on MF

Reply:
sorry i didn't get this. Can you please elaborate? what Is high risk for MF?

is_high_risk_client_for_mt5 in cfd-store cecks if user has only svg accounts and svg accounts are for CR clients

Reply:
@amina-deriv thanks for noticing, updated the comments 👍